data/scenarios/Challenges/_gallery/setup.sw

```diff
@@ -116,13 +124,10 @@ bust in the base's inventory increases monotonically.
 "idx" counts upwards.
 */
 def populateInventory = \baseCount. \idx.
-    
-    result <- tagmembers "bust" idx;
-    let total = fst result in
 
-    if (idx < total) {
+    if (idx < force bustCount) {
```

Comment:
What does `force` do here?

Reply:
When I first converted this scenario it was failing, and running it with `--autoplay` revealed that it was failing because the base started running its program *before* the level was set up.  The problem was that evaluating the definition of `bustCount` actually took multiple ticks, before the judge robot ever got around to executing its `instant setup`.  So I wrapped it in a delay so it wouldn't be evaluated at definition time.

I'm just realizing now, however, that this means it's going to be re-evaluated every time we `force` it (IIRC we don't have memoizing delay).  Perhaps a better idea would just be to wrap the *entire* program (including all the `def`s) in `instant`.

Reply:
For `word-search` (#999), I trapped the player behind a boulder until the playfield was set up :)  Though I think that may have been before the `instant` command was available.

https://github.com/swarm-game/swarm/blob/0c1d5f239ccdae13519108c37cade5eef3cccba5/data/scenarios/Challenges/_word-search/create-puzzle.sw#L244

Reply:
Strangely, wrapping the entire program in `instant` doesn't seem to work!  Not sure if this is a bug or what.